pipelines/template-build/template-build.yaml

```diff
@@ -188,31 +188,16 @@ spec:
       runAfter:
         - build-container
       taskRef:
-        name: get-clair-scan
+        name: clair-scan
```

Comment:
would be nice to rename the taskrun name (few lines above) get-clair-results -> clair-scan

Reply:
ack , like this suggestion